.github/workflows/tests.yml

```diff
@@ -14,13 +14,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-node@v1    
+    - uses: actions/setup-node@v1
       with:
         node-version: '10.x'
-      
-    - run: npm install   
+
+    - run: npm install
     - run: npm run lint
     - run: npm run typecheck
-    - run: npm run dynamodb-local-setup
-    - run: npm run test:ci
+    # - run: npm run dynamodb-local-setup
+    # - run: npm run test:ci
```

Comment:
are you disabling the test?

Reply:
yeah, not sure if the scripts above are still in use not